### PR TITLE
Load unread messages for Admin app

### PR DESCRIPTION
**Pull request by patrixr**
Since we're loading a lot less data in the admin app now, we're manually fetching the records that we need to populate the UI. More of these will probable come later

Note: will merge asap for testing

### CODE REVIEW

File: app/mixins/preload_data.js [hunk 1]
```diff
@@ -1,48 +1,82 @@
...
 
 export default Ember.Mixin.create({
-
-  preloadData: function(includePublicTypes) {
+  preloadData(includePublicTypes) {
     var promises = [];
     var isDonorApp = this.get("session.isDonorApp");
 
-    var retrieve = types => types.map(type => this.store.findAll(type, { reload: true }));
+    var retrieve = types =>
```
Comment (patrixr) [inline app/mixins/preload_data.js]: @Darshan-Chauhan can we configure the `prettier` plugin ? I don't agree with all of its decisions 😆 
I admin it's a bit frustrating when a pull request has changes that I haven't made haha

Comment (Darshan-Chauhan) [reply app/mixins/preload_data.js]: I agree that it's been frustrating 😄  .... I think if we'd used it from beginning than we wouldn't have this issue. 😅 
Should work fine once every file is done.

Comment (patrixr) [reply app/mixins/preload_data.js]: @Darshan-Chauhan where is it configured ? So that I can match the configuration to my vscode prettier plugin : )

Comment (Darshan-Chauhan) [reply app/mixins/preload_data.js]: @patrixr , it's not configured. You can add configuration if you want :) => https://prettier.io/docs/en/configuration.html

**Pull request merged**